src/firmware_variables/variables.py

```diff
@@ -86,3 +89,65 @@ def delete_variable(name, *args, **kwargs):
     :param attributes: @see Attributes
     """
     set_variable(name, value=b"", *args, **kwargs)
+
+
+class NTError(OSError):
```

Comment:
all other functions in this file throws WinError, throw WinError or inherit from WinError

You can use RtlNtstatusToDosError if you want